testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeRedirect.java

```diff
@@ -865,6 +847,26 @@ public void testHiveToDeltaPropertiesRedirect()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testHiveToDeltaPartitionsRedirect()
+    {
+        String tableName = "test_redirect_to_delta_partitions_" + randomNameSuffix();
+
+        onDelta().executeQuery(createTableOnDelta(tableName, true));
+
+        List<Row> expected = List.of(
+                row(rowBuilder().addField("regionkey", 1L).build(), 3L, 4711L),
+                row(rowBuilder().addField("regionkey", 0L).build(), 1L, 1322L));
+
+        try {
+            assertThat(onTrino().executeQuery(format("SELECT * FROM delta.default.\"%s$partitions\"", tableName))).containsOnly(expected);
+            assertThat(onTrino().executeQuery(format("SELECT * FROM hive.default.\"%s$partitions\"", tableName))).containsOnly(expected);
+        }
+        finally {
+            dropDeltaTableWithRetry("default." + tableName);
```

Comment:
Use `onDelta` method. `testHiveToDeltaPartitionsRedirect` is for Glue environment.

Reply:
Which call should use `onDelta`? This test is based off of `TestHiveAndDeltaLakeRedirect#testHiveToDeltaPropertiesRedirect`.

Reply:
Should both `onTrino` calls be replaced with `onDelta` calls? This test is based off of `TestHiveAndDeltaLakeRedirect#testHiveToDeltaPropertiesRedirect` which use `onTrino` calls.

Reply:
No, please change only dropDeltaTableWithRetry line:
```java
            dropDeltaTableWithRetry("default." + tableName);
↓
            onDelta().executeQuery("DROP TABLE default." + tableName);
```